mu-plugins/central-hub/tests/phpunit/fixtures/config-with-params-only.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ *  Test data for the ConfigStore API.
+ *  Store key is empty and configuration parameters exist.
+ *
+ * @package    spiralWebDb\centralHub\Tests\Fixtures
+ * @since      1.0.0
+ * @author     Robert A. Gadon
+ * @link       https://github.com/rgadon107/cornerstone
+ * @license    GNU General Public License 2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Fixtures;
+
+return  [
+		'aaa' => 'bbb',
```

Comment:
@rgadon107 The configuration parameters should be an array in an array.  Right?  The structure of this file should be:

```php
[
   'store_key' => [
         // config parameters
   ],
]
```

Using that structure, this file should be:

```php

```php
return [
	[
		'aaa' => 'bbb',
		'ccc' => 'ddd',
	],
];
```

Does that make sense?